app/point.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+import math
+
+
+class Point:
+    def __init__(self, cor_x: int, cor_y: int) -> None:
+        self.x = cor_x
+        self.y = cor_y
+
+    @classmethod
+    def create_from_location(cls, location: list[int]) -> Point:
+        return cls(cor_x=location[0], cor_y=location[1])
```

Comment:
To prevent `IndexError` exceptions, consider checking if the `location` list has the required number of elements before accessing its indices. You can use a condition like `if len(location) >= 2` to ensure that both `location[0]` and `location[1]` are accessible. This will make the code more robust against lists with insufficient elements.